src/Widget.svelte

```diff
@@ -2,11 +2,11 @@
 
 <script>
     import {onMount} from 'svelte';
-    import {config, data} from './stores.js';
+    import {config, cases, news} from './stores.js';
     import {ICON_DEFS} from './global.js';
-    import {dataKeyFromConfig, getElapsedTime} from './util';
+    import {getElapsedTime, hostNameFromUrl} from './util';
 
-    export let area = 'GER';
+    export let area = 'DEU';
```

Comment:
Why did you switch from `GER` to `DEU`?

Reply:
the API uses this ISO format (https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) for their endpoints: https://documenter.getpostman.com/view/2568274/SzS8rjbe?version=latest#documentation

Reply:
Don't understand that either. The correct label would be 'DE' instead of 'DEU'.

Reply:
@SimonPhumin this is a setting rather than a label. The API where we get the case data from uses the 3-letter format defined by ISO 3166-1 (see https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes)
The full API Endpoint looks like this: `https://covidapi.info/api/v1/country/DEU/latest` using anything else than `DEU` would not work for using the API.